app/presenters/keyword_presenter.rb

```diff
@@ -12,6 +12,18 @@ def matching_adword_urls
     return unless defined?(filter_params[:adwords_url_contains]) && filter_params[:adwords_url_contains]
 
     keyword.ads_top_urls.select { |item| item.downcase.include?(filter_params[:adwords_url_contains].downcase) }
+  rescue NoMethodError
+    nil
+  end
+
+  def matching_result_urls
+    return unless filter_params[:word] && filter_params[:match_at_least]
```

Comment:
```suggestion
    return if filter_params[:word].blank? || filter_params[:match_at_least].blank?
```

Composed `unless` can quickly become: 🤯 . So when more than one precicate, let's use a normal `if` instead :)